javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+var $contact = document.querySelector('#contact-form');
+$contact.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var $form = {
+    name: document.getElementById('contact-form').name.value,
+    email: document.getElementById('contact-form').email.value,
+    message: document.getElementById('contact-form').message.value
```

Comment:
You should be using the form's `elements` property to access these values. Check out step #11